src/gsi/control_vectors.f90

```diff
@@ -385,6 +387,19 @@ subroutine init_anacv
    else
       an_amp0(ii)=one/three
    endif
+   if(itracer.gt.10) then
+      if(ilev.gt.1) then
+         idaen3d(nc3d)=2
+      else
+         idaen2d(nc2d)=2
+      endif
+   else
+      if(ilev.gt.1) then
+         idaen3d(nc3d)=1
+      else
+         idaen2d(nc2d)=1
```

Comment:
While running the debug executable, the RTMA test fails with the following output:

```
forrtl: severe (408): fort: (3): Subscript #1 of the array IDAEN2D has value 0 which is less than the lower bound of 1

Image              PC                Routine            Line        Source
gsi.x              0000000007602A70  Unknown               Unknown  Unknown
gsi.x              0000000000692136  control_vectors_m         400  control_vectors.f90
gsi.x              0000000000412C6B  gsimod_mp_gsimain        1625  gsimod.F90
gsi.x              00000000004121C5  MAIN__                    618  gsimain.f90
gsi.x              000000000041209E  Unknown               Unknown  Unknown
libc-2.17.so       00002B0E900BA555  __libc_start_main     Unknown  Unknown
gsi.x              0000000000411FA9  Unknown               Unknown  Unknown
```

Printing out values for nc2d shows that this value is 0, leading to the subscript error above.  This doesn't make sense to me, because the fix/anavinfo_rtma file contains a section for control vectors.  Further investigation will be required to see what might be happening here with the RTMA.